polynote-frontend/polynote/messaging/receiver.ts

```diff
@@ -134,6 +136,21 @@ export class NotebookMessageReceiver extends MessageReceiver<NotebookState> {
                 return NoUpdate
             }
         });
+        this.receive(messages.GoToDefinitionResponse, (s, reqId, location, source) => {
+            if (s.requestedDefinition) {
+                const definition: Definition = location.map(location => {
+                    const range: IRange = {startLineNumber: location.line, endLineNumber: location.line, startColumn: location.column, endColumn: location.column};
```

Comment:
if you export `range` from `common.ts` you could use it here

Reply:
Good point!

Reply:
Moved it into `util/helpers`